### PR TITLE
Update Language List

### PR DESCRIPTION
**Pull request by ThatElfGuy**
Added Polish, Chinese and Russian to list of languages.

### CODE REVIEW

**Comment by ThatElfGuy** [comment]
Compare to the live list here: https://fetlife.com/languages

**Comment by ellie-switch** [comment]
@ThatElfGuy 
Suggested edit for last sentence:
You can also change your Language setting using your Menu options, located in the upper right of your screen.

File: 01_general/01_change-languages.md [hunk 1]
```diff
@@ -11,10 +11,13 @@ Currently, FetLife is available in the following languages:
 - Italiano
 - Magyar
 - Nederlands
+- Polski
 - Portugu&ecirc;s
 - Slovensky
 - &Epsilon;&lambda;&lambda;&eta;&nu;&iota;&kappa;ά
 - Български
+- Русский
+- 简体中文
```
Comment (johnbaku) [inline 01_general/01_change-languages.md]: Just after you deploy this, make sure that this works. If you look at the Greek one it's using HTML entities but not sure why exactly. Just something to look out for.

Maybe @reneklacan knows why?

Comment (ThatElfGuy) [reply 01_general/01_change-languages.md]: I'll keep an eye on it after merge, and revert if I need to - if it needs a revert, I'll fix it tomorrow.

**Pull request merged**